repository/r.json

```
@@ -499,6 +499,19 @@
 				}
 			]
 		},
+		{
+
+                        "name": "React + Js Snippets",
```

```suggestion
                        "name": "React + Js Snippets",
                        "previous_names": ["React​JS Snippets"],
```

If you apply this suggestion, you can then proceed to remove the entry for ReactJS Snippets [further down in this file](https://github.com/wbond/package_control_channel/pull/9003/files#diff-e0f720be39feecab629c16002d2700e6d2cad8692b9771d966eb73e7afe5a10dR647).

I hope @saada doesn't mind. We'll give them 2 weeks to object to the replacement of their package.